tests/test_settings.py

```diff
@@ -100,11 +99,6 @@
 ENABLE_TWO_FACTOR_AUTHENTICATION = False
 FORCE_TWO_FACTOR_AUTHENTICATION = False
 
-# Tell nose to measure coverage on the 'foo' and 'bar' apps
-NOSE_ARGS = ["--with-coverage", "--nologcapture", "--cover-package=arches", "--verbosity=1", "--cover-erase", "--cover-xml", "-s"]
-
-INSTALLED_APPS = INSTALLED_APPS + ("django_nose",)
-
 DATATYPE_LOCATIONS.append("tests.fixtures.datatypes")
 ELASTICSEARCH_HOSTS = [{"scheme": "http", "host": "localhost", "port": ELASTICSEARCH_HTTP_PORT}]
```

Comment:
I had to manually add "scheme": "http" over to settings_docker.py to have the tests run on Docker.